Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderIndex.spec.json

```diff
@@ -99,6 +105,16 @@
                         "items": {
                             "$ref": "#/components/schemas/Node"
                         }
+                    },
+                    "diffs": {
```

Comment:
I'm struggling to come up with a better alternative but `diffs` seems a little too generic to me... What do you think?

Maybe `versionChanges` or `versionDifferences`?

Reply:
I agree and I’m also struggling with a better alternative (`previousNodeDifferences`? 🤨)--I like `versionDifferences`, I just want to be careful that something like `versionDifferences` isn’t easily confused with the `versions` that contains patches (maybe this means the `versions` property name should be changed to be more specific?) Or maybe it won’t be too confusing since they will show up in different contexts?